en/docs/getting-started/about-this-release.md

```diff
@@ -2,113 +2,108 @@
 
 ### WSO2 API-M related new features
 
-- **[Integrating AWS Lambda with APIM]({{base_path}}/learn/tutorials/create-and-publish-awslambda-api)**
+* **[Integrating AWS Lambda with APIM]({{base_path}}/learn/tutorials/create-and-publish-awslambda-api)**
 
-     When using AWS Lambda to create and publish APIs, you can execute your code without having to manage or provision servers.
+    When using AWS Lambda to create and publish APIs, you can execute your code without having to manage or provision servers.
 
+* **[API Security Audit Integration]({{base_path}}/learn/api-security/configuring-api-security-audit)**
 
-- **[API Security Audit Integration]({{base_path}}/learn/api-security/configuring-api-security-audit)**
+    WSO2 API-M has partnered with [42Crunch](https://42crunch.com/), which is the only enterprise API security platform, in order to be able to conduct a security audit on the OpenAPI Specification definition and to obtain an audit report.
 
-     WSO2 API-M has partnered with [42Crunch](https://42crunch.com/), which is the only enterprise API security platform, in order to be able to conduct a security audit on the OpenAPI Specification definition and to obtain an audit report.
+* **[API Categories]({{base_path}}/develop/customizations/customizing-the-developer-portal/customize-api-listing/categorizing-and-grouping-apis/api-category-based-grouping)**
 
-- **[API Categories]({{base_path}}/learn/consume-api/customizations/customizing-the-developer-portal/customize-api-listing/categorizing-and-grouping-apis/api-category-based-grouping)**
+    API categories are similar to Tag wise grouping. However, admins need to define API categories that Publishers can use when creating APIs. Furthermore, unlike Tag wise grouping API categories do not use a naming convention. Therefore, with the introduction of this feature, WSO2 has deprecated the Tag wise grouping feature.
 
-     API categories are similar to Tag wise grouping. However, admins need to define API categories that Publishers can use when creating APIs. Furthermore, unlike Tag wise grouping API categories do not use a naming convention. Therefore, with the introduction of this feature, WSO2 has deprecated the Tag wise grouping feature.
+* **[AI-based recommendations for the Developer Portal]({{base_path}}/learn/consume-api/discover-apis/api-recommendations)**
 
-- **[AI-based recommendations for the Developer Portal]({{base_path}}/learn/consume-api/discover-apis/api-recommendations)**
+    The AI-based recommendations for the Developer Portal use a specialized recommendation engine to process recommendation related events and produce recommendations.
 
-     The AI-based recommendations for the Developer Portal use a specialized recommendation engine to process recommendation related events and produce recommendations. 
+* **[API Mocking]({{base_path}}/learn/design-api/mock-api/create-a-mock-api-with-an-inline-script)**
 
```

Comment:
same comment to all above as above mentioned.

Reply:
Will revert them back to one of the supported list types :)